packages/react/src/List/ListItem.tsx

```diff
@@ -0,0 +1,40 @@
+import React, {type Ref, useContext} from 'react'
+import clsx from 'clsx'
+import {CheckIcon, DashIcon} from '@primer/octicons-react'
+import type {BaseProps} from '../component-helpers'
+import {ListContext} from './UnorderedList'
+
+import styles from './List.module.css'
+
+export type ListItemProps = BaseProps<HTMLElement> & {
+  /**
+   * React.ReactNode and React.ReactNode[] are valid children.
+   */
+  children?: React.ReactNode | React.ReactNode[]
```

Comment:
You can use `PropsWithChildren` instead of manually defining children like this. Should be examples in other components. Maybe AnchorNav.

Reply:
When I use this I have run into the following error:
```
Type '{ children: Element; accessKey: string; accessKeyLabel: string; autocapitalize: string; dir: string; draggable: boolean; hidden: boolean; inert: boolean; innerText: string; lang: string; ... 277 more ...; className: string; }' is not assignable to type 'DetailedHTMLProps<HTMLAttributes<HTMLUListElement>, HTMLUListElement>'.
  Type '{ children: Element; accessKey: string; accessKeyLabel: string; autocapitalize: string; dir: string; draggable: boolean; hidden: boolean; inert: boolean; innerText: string; lang: string; ... 277 more ...; className: string; }' is not assignable to type 'HTMLAttributes<HTMLUListElement>'.
```

Reply:
Fixed in next push